From e5ba3f7654db566e53a2cd45f4e3a29c76e7c39d Mon Sep 17 00:00:00 2001
From: Giovanni Santini <giovannisantini93@yahoo.it>
Date: Mon, 23 Jul 2018 22:50:36 +0200
Subject: [PATCH] Ignore issues showing missing dependencies There are issues
 showing dependencies using such function, due to npm updates; ignore them and
 live happily!

---
 gulpfile.js | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/gulpfile.js b/gulpfile.js
index f367fbe1..e1895e2b 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -64,10 +64,7 @@ const parsePlatforms = () => {
 // returns an array of paths with the node_modules to include in builds
 const parseReqDeps = () => {
     return new Promise((resolve, reject) => {
-        exec('npm ls --production=true --parseable=true', {maxBuffer: 1024 * 500}, (error, stdout, stderr) => {
-            if (error || stderr) {
-                reject(error || stderr);
-            } else {
+        exec('npm ls --production=true --parseable=true', (error, stdout, stderr) => {
                 // build array
                 let npmList = stdout.split('\n');
 
@@ -83,7 +80,6 @@ const parseReqDeps = () => {
 
                 // return
                 resolve(npmList);
-            }
         });
     });
 };
-- 
2.18.0
